config/application.yml.example

```diff
@@ -16,6 +16,7 @@ development:
   WORKER_HOST: http://localhost:3002
   WORKER_API_KEY: workerkey
   API_HOST: http://localhost:3000
+  HARVESTER_API_KEY: "provide Supplejack API TOKEN with HARVESTER previleges"
```

Comment:
Do we need a specific Harvester Key or can we use the Worker key and give it harvester privileges?

Reply:
The role needs to have the attribute `harvester: true`